1/st.labs2017/st.labs2017/index.html

```diff
@@ -0,0 +1,391 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <title></title>
+    <meta charset="utf-8" />
+    <link href="/content/css/main.css" rel="stylesheet" />
+</head>
+<body>
+    <!-- START HEADER -->
+    <header class="header">
+        <div class="container">
+            <div class="header__logo">
```

Comment:
This element should care about positioning and .logo should be a separate block.